nrf_security/configs/nrf-config.h.template

```diff
@@ -2168,7 +2168,7 @@
  *            it, and considering stronger ciphers instead.
  *
  */
-#define MBEDTLS_ARC4_C
+#cmakedefine MBEDTLS_ARC4_C
```

Comment:
`#cmakedefine` should only be used for settings we control using Kconfig + CMake.

I don't see ARC4 being configurable through Kconfig / CMake.

Reply:
This has been fixed with adding section in cmake